Documentation/Helm-Charts/operator-chart.md

```diff
@@ -133,6 +133,10 @@ The following tables lists the configurable parameters of the rook-operator char
 | `admissionController.tolerations`   | Array of tolerations in YAML format which will be added to admission controller deployment.                                                      | `<none>`                                                  |
 | `admissionController.nodeAffinity`  | The node labels for affinity of the admission controller deployment [^1]                                                                         | `<none>`                                                  |
 | `monitoring.enabled`                | Create necessary RBAC rules for Rook to integrate with Prometheus monitoring in the operator namespace. Requires Prometheus to be pre-installed. | `false`                                                   |
+| `csi.csiRBDPluginVolume`            | The volume of the CephCSI RBD plugin DaemonSet [^1]                                                                                | `<none>`                                                  |
```

Comment:
The footnote doesn't apply to these, right?
```suggestion
| `csi.csiRBDPluginVolume`            | The volume of the CephCSI RBD plugin DaemonSet                                                                                | `<none>`                                                  |
```

Reply:
Yes, correct, it doesn't apply here, correct it.